ext/storage_sources/azure_store/README.md

```diff
@@ -0,0 +1,106 @@
+# WiredTiger's Azure Extension
+## 1. Introduction
+This extension allows WiredTiger storage source extensions to read from and write to objects stored in Azure Blob Storage using WiredTiger’s provided internal abstraction for storing data in an object storage service.
+
+## 2. Building and running
+This section describes how to build WiredTiger with the Azure extension enabled.
+
+### Requirements
+<li> CMake 3.13 or higher
+<li> G++ 8.4 or higher <p>
+
+Check your CMake version by typing `cmake --version`.
+Check your g++ version typing `g++ --version`.
+
+### How to install requirements (skip this step if requirements have been met)
+
+If the CMake version is not 3.13 or higher update your CMake to 3.13 using the following.
+```bash
+sudo apt remove cmake
+wget https://cmake.org/files/v3.13/cmake-3.13.0.tar.gz
+tar xf cmake-3.13.0.tar.gz
+
+cd cmake-3.13.0
+
+./configure
+make -j $(nproc)
+sudo make install
+```
+
+Check that your CMake has been updated using the following command `cmake --version`.
+
+If your cmake is not in `/usr/bin/` create a symbolic link using the following.
+```bash
+sudo ln -s /usr/local/bin/cmake /usr/bin/cmake
+```
+
+If the G++ version is not 8.4 or higher update your G++ to 8.4 using the following.
+```bash
+sudo apt-get install gcc-8 g++-8
+sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 20 --slave /usr/bin/g++ g++ /usr/bin/g++-8
+```
+
+Check that your G++ has been updated using the following command `g++ --version`.
```

Comment:
I'm not a huge fan of this section. The `sudo ln -s` line can be OK for some individual making changes to a specific system, but it's not suitable as general advice. I think we can probably lose this section entirely - it's preferable to have the system package manager install/manage these dependencies anyway, and different systems will all use different tooling (APT, yum, pacman, etc.)

Reply:
i've removed the whole instruction on how to install the requirements in my latest commit

Reply:
Thanks